channels-rke2.yaml

```diff
@@ -615,6 +615,20 @@ releases:
         version: 0.1.1300
     featureVersions: &featureVersions-v1
       <<: *featureVersions-v1-22-11-rke2r1
+  - version: v1.22.14-rc1+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v1
+    agentArgs: *agentArgs-v1
+    charts: &charts-v2
```

Comment:
We have `charts-v2` defined already on line 752, so we should name this something different. Maybe what we did previously was confusing... I'm thinking we should update the charts versions to:
`charts-v122-v1`, `charts-v123-v1`, etc. so that it's easier and more clear going forward. Or maybe just need to do it for v122 since it's different enough from 1.23+ at this point?